src/inspector/view/qml/MuseScore/Inspector/general/appearance/internal/VerticalSpacingSection.qml

```diff
@@ -33,7 +33,7 @@ SpinBoxPropertyView {
     property PropertyItem minimumDistance: null
 
     anchors.left: parent.left
-    anchors.right: parent.horizontalCenter
+    anchors.right: parent.right
     anchors.rightMargin: 2
```

Comment:
I think you would also need to remove the `anchors.rightMargin` (also in HairpinStyleSettings.qml), because now it doesn't line up perfectly:
<img width="1548" alt="Schermafbeelding 2021-11-29 om 11 41 38" src="https://user-images.githubusercontent.com/48658420/143853600-72634351-352c-4d37-9c67-c9be88ab2b66.png">
(That margin was there to make it line up with the PropertyViews above it, which had some spacing between them.)

Reply:
Very astute observation, @cbjeukendrup!